plugins/sql/Cargo.toml

```
@@ -29,10 +29,11 @@ tauri = { workspace = true }
 log = { workspace = true }
 thiserror = { workspace = true }
 futures-core = "0.3"
-sqlx = { version = "0.8", features = ["json", "time"] }
+sqlx = { version = "0.8", features = ["json", "time", "uuid"] }
 time = "0.3"
 tokio = { version = "1", features = ["sync"] }
 indexmap = { version = "2", features = ["serde"] }
+uuid = "1.19.0"
```

```suggestion
uuid = "1"
```

I'll try to set loose minimum versions if possible.

---

Done